realm/realm-library/build.gradle

```diff
@@ -46,6 +46,7 @@ repositories {
 
 dependencies {
     provided 'io.reactivex:rxjava:1.1.0'
+    compile "com.android.support:recyclerview-v7:23.1.1"
```

Comment:
With provided I have got `java.lang.NoClassDefFoundError: io.realm.entities.RealmRecyclerAdapter` in tests. With added `androidTestCompile 'com.android.support:recyclerview-v7:23.1.1'` I always get `Error:Cannot add task ':realm-library:prepareComAndroidSupportRecyclerviewV72311Library' as a task with that name already exists.` Could you verify this?